static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -414,13 +423,13 @@ $navigation-horizontal-padding: 38px;
 }
 
 .main-navigation__icon {
-    top: .04em;
+    top: .14em;
```

Comment:
Magic they are but cos it's a square rotated it needs to be optically aligned and can't be done any other way. At least it will grow with the font size 😄